docs/quick-start.md

```diff
@@ -0,0 +1,220 @@
+# gRPC Swift Quick Start
+
+## Before you begin
+
+### Prerequisites
+
+#### Swift Version
+
+gRPC requires Swift 5.0 or higher.
+
+#### Install Protocol Buffers v3
+
+Install the protoc compiler that is used to generate gRPC service code. The
+simplest way to do this is to download pre-compiled binaries for your
+platform (`protoc-<version>-<platform>.zip`) from here:
+[https://github.com/google/protobuf/releases][protobuf-releases].
+
+* Unzip this file.
+* Update the environment variable `PATH` to include the path to the `protoc`
+  binary file.
```

Comment:
", or move the `protoc` binary into a directory that is part of your `PATH`"?

Reply:
I'd rather keep the wording as it is: this is the wording for all of the quick start guides on grpc.io